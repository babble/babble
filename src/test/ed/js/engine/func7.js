
print( null ==
       function(){
       }() 
     );

print( null ==
       function(){
           a = 5;
       }() 
     );

print( null ==
       function(){
           var a = 5;
       }() 
     );

print( null ==
       function(){
           var a = 5;
           a += 3;
       }() 
     );

print( 
    function(){
        var a = 5;
        a += 3;
        return a;
    }() 
);
