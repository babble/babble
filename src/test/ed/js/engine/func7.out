true
true
true
true
8
